blocked-edges/4.14.33-SRIOVFailedToConfigureVF.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.33
+from: ^4[.](13[.].*|14[.]([0-9]|1[0-9]|2[0-9]|3[0-2]))[+].*$
+url: https://issues.redhat.com/browse/NHE-1171
+name: SRIOVFailedToConfigureVF
+message: |-
+  Pods using a secondary interface of SRI-OV VF will stop coming up.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
```

Comment:
nit: no need for the wrapping parens, because there's nothing else that we're grouping against.

Reply:
Fixed in c5b088415a6e9bd0 -> 1dddc4dec09c2acd94; I'm marking this thread resolved.

Reply:
Done.